FLEX.podspec

```diff
@@ -41,7 +41,7 @@ Pod::Spec.new do |spec|
        'CLANG_CXX_LANGUAGE_STANDARD' => 'gnu++11',
   }
   spec.compiler_flags   = "-Wno-unsupported-availability-guard", "-Wno-deprecated-declarations"
-  spec.public_header_files = [ "Classes/*.h", "Classes/Manager/*.h", "Classes/Toolbar/*.h",
+  spec.public_header_files = [ "Classes/**/*.h", "Classes/Manager/*.h", "Classes/Toolbar/*.h",
```

Comment:
Doesn't this mean develoeprs will be able to depend on public classes which ideally they shouldn't have access to?

Because the author probably purposeful has hidden them from external users?

Reply:
Yes, that's right mas, with this, all headers would be publicly available to access. 

For our use-case we are anyway consuming it in Swift so we import it as a module than header files, If anyone tries to invoke a method which is not meant to be invoked then it might be a problem.

Reply:
Can another approach be to modify the import statements without having to move the public header files around?

I believe that should be even more minimal changes compared to moving files around.

Mostly thinking in terms of how can we make it for future contributors to keep this fork up to date.

Reply:
> Can another approach be to modify the import statements without having to move the public header files around?

What exactly we need to modify here? 🤔 I mean the import statements stay the same right? 

OR you mean to say using full path imports like this ? `#import "Module/directory/header.h"` ?

Reply:
> OR you mean to say using full path imports like this ?

Yes this approach mas.